orderedwriter/orderedwriter_test.go

```diff
@@ -15,6 +15,16 @@ const (
 	testRuns = 64
 )
 
+func randomBytes(n int) []byte {
```

Comment:
i meant, you can use `randomString` and convert it to byte slice :)

Reply:
Since the function you have mentioned is under the package `e2e` and this test is under `orderedwriter`, I thought rather than importing it, copying would be better. Also avoiding unnecessary casting seemed better too :)

Reply:
it's a test helper, so string->[]byte conversion cost is negligible. it might be better to create a new `testutil` package to store test helpers but it's not necessary for this PR.